README.md

```diff
@@ -12,56 +12,70 @@ Since the very beginning, the Ada language has been formally defined through a p
+
+At the present time, AIs are being stored in the Rich Text Format, with an aim to eventually develop tooling to easily convert it into many formats. RTF was selected due to its maturity, stability, and generality. The ARG will continue to review the AI formats in the future.
+
+An AI template can be found in the root of this repository.
+
+# A new way to engage the Ada user community
+
+Historically the ada-comment mailing list has been used by members of the Ada community to identify problems or suggest enhancements for consideration by the ARG.  
 
 This repository, in conjunction with a new [website](https://arg.adaic.org/home), has been created to simplify the process of providing Ada community input to the ARG, and to provide more visibility into the ARG process.
 
 ## An accessible medium for discussion
 
 This repository, when compared with the sister [website](https://arg.adaic.org/home), is geared towards supporting real-time discussions on proposed features or changes to the Ada language through Github's Issues feature.
 
+Users that have comments or proposals are encouraged to open a new issue in this repository to drive initial discussion, including input from members of the ARG.
+
 Active issues can be browsed and discussed with broad accessibility and transparency.
 
+## Getting started
 
-# Language Study Groups
+The best way to get started is to browse open issues to get a sense of how issues are structured, and to see if you find any existing issues that interest you. Consider speaking with supporters within that discussion about proposing a Language Study Group to take the issue further.
 
-Language Study Groups can be formed by the community to work transparently towards a formal proposal (Ada Interpretation/AI) for consideration by ISO/IEC JTC 1/SC 22/WG 9's Ada Rapporteur Group (ARG).
+If there is an issue that you have not found discussed, please feel welcome to open a new issue. Try to give it a concise title, a succinct description, and an appropriate tag.
 
-It is encouraged that issues first be put to discussion as a Github Issue in this repository, to get feedback from the community, and possibly to identify any early issues, before forming a Study Group. 
+If you find an issue is not gaining enough traction, or has received no comments after a month, considering "pinging" the issue generate notifications to anyone following the issue or the repository.
 
-Anyone who wishes to form a Language Study Group to work towards a formal AI should submit a request through the official ARG [web-form](https://arg.adaic.org/community-input).
+For issues that gain significant traction, it may be encouraged to propose the creation of a Language Study Group to develop the idea further, and ultimately into an AI. See the Language Study Groups section below.
```

Comment:
"... it may be encouraged ..." ==> "... you are encouraged ..." or "... you might be encouraged ..."

Reply:
Let's not be wishy-washy. "you are encouraged" is best.

Reply:
Tucker suggested in Issue #48 to add something here (or somewhere) noting that Issues should be closed by the management only (not by the creator or other ARG members). And when you do, you can close that Issue.